modules/sr/robot/motor_devices.py

```diff
@@ -13,6 +13,12 @@ def __init__(self, webot: Robot, motor_name: str) -> None:
         self.webot_motor = webot.getMotor(motor_name)
         self.max_speed = self.webot_motor.getMaxVelocity()
 
+        # Apply the brake when this motor speed = 0 by default
+        self._use_brake = True
+
+    def set_brake(self, value: bool) -> None:
```

Comment:
```suggestion
    def _set_brake(self, value: bool) -> None:
```
As we are not encouraging students to call this function directly, we should make it 'private' by prefixing the function name with an underscore.

Reply:
I disagree -- this class is already not part of the public API anyway, so I think it's more useful that we consider public/private at the level of "should this be called from outside the current class". In that context I think the current spelling is preferable.

If anything, I would encourage that `self._use_brake` be renamed to `self.use_brake` and just assigned to/read from directly (since we're already reading it directly).